src/components/Strategies/StrategyList.tsx

```diff
@@ -28,7 +30,13 @@ function StrategyList() {
       if (strategy.comingSoon) {
         comingSoonText!.classList.add('visible');
       } else {
-        history.push('/strategies/' + strategy.url);
+        if (user.apiKey) {
```

Comment:
Nested if. Just declare another function that you will call here and put this logic inside that